packages/core/src/RenderingEngine/Viewport.ts

```diff
@@ -589,15 +589,15 @@ class Viewport implements IViewport {
     const { storeAsInitialCamera } = displayArea;
 
     // make calculations relative to the fitToCanvasCamera view
-    this.setCamera(this.fitToCanvasCamera, storeAsInitialCamera);
+    this.setCamera(this.fitToCanvasCamera, false);
```

Comment:
can you please add a comment why we are changing it? it was previously storeAsInitialCamera